src/actinia_core/core/geodata_download_importer.py

```diff
@@ -215,22 +215,39 @@ def get_file_rename_command(file_path, file_name):
         return p
 
     @staticmethod
-    def get_raster_import_command(file_path, raster_name):
+    def get_raster_import_command(file_path, raster_name, resample=None,
+                                  resolution=None, resolution_value=None):
         """Generate raster import process list that makes use of r.import
 
         Args:
-            file_path:
-            raster_name:
+            file_path (str): The path of the raster file to import
+            raster_name (str): The name of the new raster layer
+            resample (str): The resampling method to use for reprojection
+                            Options: nearest, bilinear, bicubic, lanczos, bilinear_f,
+                            bicubic_f, lanczos_f (default: nearest)
+            resolution (str): The resolution of the new raster layer
+                              Options: estimated, value, region (default: estimated)
+            resolution_value (str): Resolution of output raster map (use with option
+                                      resolution=value)
 
         Returns:
             Process
 
         """
+        executable_params = ["input=%s" % file_path,
+                             "output=%s" % raster_name,
+                             "--q"]
+
+        if resample is not None:
+            executable_params.append("resample=%s" % resample)
```

Comment:
Would there be any benefit in checking if `resample` has one of the allowed values `"nearest", "bilinear"`, etc.? This is not the case right now, and I reckon then `r.import` fails with a proper error message. It might still be worth it, because then the process fails directly and not when it comes to the importing step (although all importing steps are performed at the beginning of the PC)

Reply:
checks have been added in commit [66de85f3bf17dbe048b22336fa07ec6037f5653f](https://github.com/mundialis/actinia_core/pull/298/commits/66de85f3bf17dbe048b22336fa07ec6037f5653f)